gobblin-metrics/src/main/java/gobblin/metrics/KafkaAvroReporter.java

```diff
@@ -0,0 +1,163 @@
+/* (c) 2014 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.metrics;
+
+import com.codahale.metrics.MetricRegistry;
```

Comment:
Please order imports in the following order:

1. Built in java classes.
2. org classes.
3. com classes.
4. Gobblin classes.